dataclients/kubernetes/kube.go

```diff
@@ -279,14 +317,49 @@ func (c *Client) loadAndConvert() ([]*eskip.Route, error) {
 	return r, nil
 }
 
+func healthcheckRoute(healthy bool) *eskip.Route {
+	status := http.StatusOK
+	if !healthy {
+		status = http.StatusNotFound
```

Comment:
I'm a bit unsure if it shouldn't be a 5XX family error code. Like `503 - Service Unavailable`.
4XX are client errors and this is not the case. In practical terms it serves the same purpose.
What do you think?